src/components/common/tag.jsx

```diff
@@ -33,11 +36,71 @@ const Tags = styled.ul`
   padding: 0;
 `
 
+export class TagListCutter extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { allOpen: false }
+  }
+  handleChange() {
+    this.setState({ allOpen: !this.state.allOpen })
+  }
+  render() {
+    const { tagNumberDesktop, tagNumberMobile, tags } = this.props
+
+    return (
+      <Media.MobileOnly>
+        {isMobile => {
+          let currentNumber = isMobile ? tagNumberMobile : tagNumberDesktop
+          return currentNumber && currentNumber < tags.length
+            ? this.props.children(
+                this.state.allOpen ? tags : tags.slice(0, currentNumber),
+                ToogleLabel(() => this.handleChange(), this.state.allOpen),
+                MoreTag(() => this.handleChange(), this.state.allOpen),
+              )
+            : this.props.children(tags)
+        }}
+      </Media.MobileOnly>
+    )
+  }
+}
+
+const ToogleLabel = (onLableClick, allOpen) => (label, theme) => (
+  <div style={{ display: 'flex' }}>
+    <TagListLabel>{label}</TagListLabel>
+    <ToogleButton onClick={onLableClick} down={!allOpen}>
+      <DownArrowIcon fill={theme.greyishBrown} />
+    </ToogleButton>
+  </div>
+)
+
+const ToogleButton = styled.div`
+  display: flex;
+  transform: ${props => (props.down ? 'rotate(0deg)' : 'rotate(180deg)')};
+  cursor: pointer;
+`
+
+const MoreTag = (onLableClick, allOpen) => () =>
+  allOpen ? null : (
+    <Tag
+      key="moreTagsHere"
+      color={morePlaqueColor}
+      onClick={() => onLableClick()}>
+      more tags...
+    </Tag>
+  )
+
 export const TagList = withTheme(
-  ({ tags, selectedTags, label, theme, onTagClick }) => (
+  ({
+    tags,
+    selectedTags,
+    label,
+    theme,
+    onTagClick,
+    LabelComponent = label => <TagListLabel>{label}</TagListLabel>,
```

Comment:
if you are using render props, it is better to name them `renderLabel`, `renderMoreTags` to make it clearer for the user of the component what should he pass in.